internal/controller/deployment_test.go

```diff
@@ -110,8 +110,10 @@ func TestConstructEnvForApp(t *testing.T) {
 		value     string
 		valueFrom *corev1.EnvVarSource
 
-		expectedOtelVars map[string]string
-		otelVars         spinv1alpha1.OtelConfig
+		expectedOtelVars             map[string]string
+		otelVars                     spinv1alpha1.OtelConfig
+		invocationLimits             map[string]string
+		expectedInvocationLimitsVars map[string]string
```

Comment:
Non-blocking nit: you could simplify this by merging `expectedOtelVars` and `expectedInvocationLimitsVars` into one map `expectedVars`